docs/user-guide.rst

```diff
@@ -363,6 +363,7 @@ Common build options
    context. Note that if the hardware supports this extension and this option is
    set to 0 the value of the registers will be leaked between Secure and
    Non-secure worlds if PAuth is used on both sides. The default is 0.
+   This is an experimental feature.
```

Comment:
Add: "Note that pointer Authentication is enabled in Non-Secure world irrespective of any build flag if the CPU supports it."